### ISSUE TITLE
Add an over-time comparison for telemetry missing columns  

### ISSUE THREAD

**Issue by relud** [mozilla/bigquery-etl#644]
we currently have a dashboard [0], based on telemetry_missing_columns_v1, which shows us the top missing columns, but what we really need is to show how this has changed over time.

[0] https://sql.telemetry.mozilla.org/queries/65797/source#167094

**Comment by acmiyaguchi** [comment]
I took a quick look at this -- I think one I the reasons this wasn't added was due to the cost of the view. The full query scans over 1.1TB a day with a significant number of intermediate rows. The resulting dataset is only 65,273 rows for a single day.

The approach I would take would be to materialize the view (scheduled daily), and then write the queries on top of this. In this way, we would be able to look over all time in a fairly inexpensive way.
